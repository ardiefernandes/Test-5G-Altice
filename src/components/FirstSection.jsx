import React from "react";
import Banner from "../assets/MEO 5G Banner.png";

const FirstSection = () => {
  return (
    <section className="first-section">
      <h1 aria-label="A evolução tecnológica que levas contigo">
        A EVOLUÇÃO tecnológica que levas contigo
      </h1>
      <img alt="meo 5G" src={Banner} />
    </section>
  );
};

export default FirstSection;
