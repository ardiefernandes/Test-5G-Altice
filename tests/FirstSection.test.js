import React from "react";
import renderer from "react-test-renderer";
import FirstSection from "../src/components/FirstSection";

it("correctly renders the page first section", () => {
  const component = renderer.create(<FirstSection />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
