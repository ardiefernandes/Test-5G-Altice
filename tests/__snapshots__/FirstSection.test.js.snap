// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly renders the page first section 1`] = `
<section
  className="first-section"
>
  <h1
    aria-label="A evolução tecnológica que levas contigo"
  >
    A EVOLUÇÃO tecnológica que levas contigo
  </h1>
  <img
    alt="meo 5G"
    src="MEO 5G Banner.png"
  />
</section>
`;
